examples/plotting/file/sprint.py

```diff
@@ -38,9 +40,11 @@
 fill_color = { "gold": gold_fill, "silver": silver_fill, "bronze": bronze_fill }
 line_color = { "gold": gold_line, "silver": silver_line, "bronze": bronze_line }
```

Comment:
As long as we are editing this file, it's pretty long. What do you think of just putting the color codes directly in these dicts and deleting the named variables above?